packages/jest-circus/package.json

```diff
@@ -22,6 +22,7 @@
     "stack-utils": "^1.0.1"
   },
   "devDependencies": {
+    "execa": "^0.10.0",
```

Comment:
we usually just use them from `devDependencies` in root

Reply:
I know, but this was pre yarn workspaces. We should eventually move all the dev deps to packages imo

Reply:
that will unnecessarily produce version bumps on dev dep changes, though

Reply:
Hm I'm keen on leaving this, because I'd need to add eslint disables. By version bumps you mean learna bumping e.g. `jest-circus` version based on dev deps? That would be weird